src/layer/dataset.ts

```diff
@@ -56,6 +64,8 @@ export const DATASET_S2L1C: Dataset = {
   searchIndexUrl: 'https://services.sentinel-hub.com/index/v3/collections/S2L1C/searchIndex',
   findDatesUTCUrl: 'https://services.sentinel-hub.com/index/v3/collections/S2L1C/findAvailableData',
   orbitTimeMinutes: 50.3,
+  minDate: new Date(Date.UTC(2015, 1 - 1, 1, 0, 0, 0)),
```

Comment:
[Docs](https://docs.sentinel-hub.com/api/latest/#/data/Sentinel-2-L1C?id=endpoint-locations) say the availability is from November 2015 (checked for January 2015, no dates available)

Reply:
Searched for it, it seems the real availability date is `2015-06-17`. Fixed, thanks!

Reply:
No problem. But not to loose too much time, let's check the others in a separate MR.

Resolving